foolbox/models/mxnet.py

```diff
@@ -63,7 +63,11 @@ def __init__(
         label = mx.symbol.Variable('label')
         self._label_sym = label
 
-        loss = mx.symbol.softmax_cross_entropy(logits, label)
+        log_softmax = mx.sym.log_softmax(logits)
```

Comment:
can you add a comment pointing out that this is a workaround for https://github.com/apache/incubator-mxnet/issues/6874